### ISSUE TITLE
Hash changed for DirectoryPage_Game persisted query

### ISSUE THREAD

**Issue by gibbed** [Windows200000/TwitchDropsMiner-updated#159]
```json
{
  "errors": [
    {
      "message": "PersistedQueryNotFound"
    }
  ],
  "extensions": {
    "durationMilliseconds": 3,
    "operationName": "DirectoryPage_Game",
    "requestID": "CENSORED"
  }
}
```

The new `sha256Hash` is `c7c9d5aad09155c4161d2382092dc44610367f3536aac39019ec2582ae5065f9`.

I don't see anything new in variables.

Reposted from [the parent repo](https://github.com/DevilXD/TwitchDropsMiner/issues/523). Someone's inevitably going to post it here too so may as well.

**Comment by gibbed** [comment]
Looks like they've finally axed the old hash.

**Comment by Windows200000** [comment]
Fix included in [v15.8.2](https://github.com/Windows200000/TwitchDropsMiner-updated/releases/tag/v15.8.2). Sorry for being so late.
